aten/src/ATen/FastPass.cpp

```diff
@@ -0,0 +1,34 @@
+// FastPass
```

Comment:
You mentioned there are some "fast_pass" function calls - do you know roughly how many of them?
Are we going to copy all their cpu implementation into this file? How is it compared to putting it in ScalarOps.cpp?

Reply:
scalar_tensor is the one referenced from lite trainer. There can be others but likely not built for mobile. It's a good idea to put it in ScalarOps.cpp!